tools/licenses/lib/filesystem.dart

```diff
@@ -11,6 +11,8 @@ import 'package:archive/archive.dart' as a;
 
 import 'cache.dart';
 
+const int kMaxSize = 5 * 1024; // only look for copyrights and licenses at the top of the file
```

Comment:
this is already defined in licenses.dart. We should definitely only have one of these.

Reply:
Done.